README.md

```diff
@@ -1,7 +1,15 @@
 [![GoDoc](https://godoc.org/github.com/containers/image?status.svg)](https://godoc.org/github.com/containers/image) [![Build Status](https://travis-ci.org/containers/image.svg?branch=master)](https://travis-ci.org/containers/image)
 =
 
-`image` is a set of Go libraries aimed at working in various way with containers' images.
+`image` is a set of Go libraries aimed at working in various way with containers' images and container image registries. 
```

Comment:
@rhatdan Travis is failing because it detected that at the end of this line there's a whitespace (just after the final dot), could you remove it and re-push?